<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapWindow</class>
 <widget class="QMainWindow" name="MapWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Conquest of Pirkanmaa</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>250</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="resourcesHeading">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>151</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Resources</string>
         </property>
        </widget>
        <widget class="QLabel" name="turnLabel">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>10</y>
           <width>181</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Player 1's turn</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QFrame" name="frame_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>321</width>
           <height>131</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="coinImg">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>30</width>
            <height>30</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Coins</string>
          </property>
          <property name="text">
           <string>Img</string>
          </property>
         </widget>
         <widget class="QLabel" name="woodImg">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>30</width>
            <height>30</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Wood</string>
          </property>
          <property name="text">
           <string>img</string>
          </property>
         </widget>
         <widget class="QLabel" name="coinLabel">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>10</y>
            <width>331</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>13 (+2)</string>
          </property>
         </widget>
         <widget class="QLabel" name="woodLabel">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>40</y>
            <width>339</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>40</string>
          </property>
         </widget>
         <widget class="QLabel" name="foodLabel">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>70</y>
            <width>341</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>3 (+0.3)</string>
          </property>
         </widget>
         <widget class="QLabel" name="foodImg">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>30</width>
            <height>30</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Food</string>
          </property>
          <property name="text">
           <string>img</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="tabBarAutoHide">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Tile</string>
         </attribute>
         <widget class="QLabel" name="tileHeaderLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>321</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Grassland</string>
          </property>
         </widget>
         <widget class="QLabel" name="tileDescriptionLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>261</width>
            <height>171</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Incredibly flat land. It's suitable for farming!</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="tileImgLabel">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>10</y>
            <width>60</width>
            <height>60</height>
           </rect>
          </property>
          <property name="text">
           <string>Img</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Building</string>
         </attribute>
         <widget class="QLabel" name="buildingHeaderLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>321</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Headquarters</string>
          </property>
         </widget>
         <widget class="QLabel" name="tileDescriptionLabel_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>221</width>
            <height>81</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Home sweet home.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="buildingImgLabel">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>10</y>
            <width>60</width>
            <height>60</height>
           </rect>
          </property>
          <property name="text">
           <string>Img</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Unit</string>
         </attribute>
         <widget class="QProgressBar" name="healthBar">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>80</y>
            <width>131</width>
            <height>23</height>
           </rect>
          </property>
          <property name="value">
           <number>69</number>
          </property>
          <property name="format">
           <string>%p</string>
          </property>
         </widget>
         <widget class="QLabel" name="unitTypeLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>211</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Type: Archers</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="unitTextBox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>261</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>Archers 1</string>
          </property>
         </widget>
         <widget class="QLabel" name="healthLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>80</y>
            <width>76</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Health</string>
          </property>
         </widget>
         <widget class="QLabel" name="movementPointLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>110</y>
            <width>201</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Movement points</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>110</y>
            <width>76</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>2</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_2">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>160</y>
            <width>100</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>Attack</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_3">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>160</y>
            <width>100</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
         <widget class="QLabel" name="unitImgLabel">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>10</y>
            <width>60</width>
            <height>60</height>
           </rect>
          </property>
          <property name="text">
           <string>Img</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>90</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="endTurnButton">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>10</y>
           <width>141</width>
           <height>71</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>End your turn after you are done.</string>
         </property>
         <property name="text">
          <string>End Turn</string>
         </property>
        </widget>
        <widget class="QLabel" name="turnCountLabel">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>30</y>
           <width>131</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Turn: 134</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QPushButton" name="zoomInButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>30</width>
           <height>30</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="toolTip">
          <string>Zoom in</string>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
        <widget class="QPushButton" name="zoomOutButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>30</width>
           <height>30</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>17</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Zoom out</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Game"/>
    <addaction name="separator"/>
    <addaction name="actionReturn_to_Main_Menu"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_Game">
   <property name="text">
    <string>Save Game</string>
   </property>
  </action>
  <action name="actionReturn_to_Main_Menu">
   <property name="text">
    <string>Return to Main Menu</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
